packages/server/docs/openapi.yaml

```diff
@@ -0,0 +1,197 @@
+openapi: 3.0.0
```

Comment:
to preview the HTML version you can open
https://editor.swagger.io/

and click `File` -> `Import URL`
paste in

`https://raw.githubusercontent.com/amites/upswyng/alvin/docs/packages/server/docs/openapi.yaml`

Reply:
Ya, I think that this is looking good to me 👍  I haven't done a lot of API documentation myself, but makes sense to use this standard. Since a good number of tools support this spec, I think this is a good base if we end up wanting something more robust. Awesome 🎉

Reply:
I like it! Cool that it generates curl examples
<img width="884" alt="Screen Shot 2021-07-02 at 10 17 24 AM" src="https://user-images.githubusercontent.com/51095001/124302676-c07f4480-db1e-11eb-900c-be491b0c8133.png">